components/cover/he60r.rst

```diff
@@ -0,0 +1,85 @@
+HE60R Garage Door Opener
+========================
+
+.. seo::
+    :description: Setting up a Chamberlain HomEntry HE60R and similar garage door openers
+    :image: he60r.jpg
+
+.. figure:: images/he60r.jpg
+    :align: right
+
+The ``he60r`` cover platform allows you to control a HomEntry HE60R
+garage door opener. These units were made by Chamberlain and sold through Bunnings. It is *likely* that this
+component will also work with other older Chamberlain garage door openers. It is not currently compatible with
+newer Chamberlain units that use the Security+ 2.0 protocol.
+
+A sample configuration entry is here (this corresponds to the sample circuit below.) Pin numbers should be specified
+to match your hardware configuration. The TX pin is inverted in this configuration since the sample circuit
+uses an inverting transistor driver to send data.
+
+.. code-block:: yaml
+
+    # Example configuration entry
+
+    uart:
+      - id: door_uart
+        baud_rate: 1200
+        parity: even
+        stop_bits: 1
+        tx_pin:
+          number: 23
+          inverted: true
+        rx_pin: 22
```

Comment:
Components should not have external config examples. That is for the user to read about and implement. Otherwise they potentially do not understand why the example they literally copied doesnt work.

For uart etc we usually have a line above the example with a reference to uart docs saying it needs to be set up.